### PR TITLE
chore: switch to uds zarf

### PR DESCRIPTION
**Pull request by mjnagel**
Removes need for zarf on its own, switches to the incredible `uds zarf` instead.

### CODE REVIEW

File: renovate.json [hunk 1]
```diff
@@ -34,29 +34,12 @@
...
-      "depNameTemplate": "defenseunicorns/zarf",
-      "fileMatch": ["README\\.md"],
-      "matchStrings": [
-        "- Zarf: (?<currentValue>.*)"
-      ],
-      "datasourceTemplate": "github-tags",
-      "extractVersionTemplate": "^v(?<version>.*)$"
-    },
-    {
-      "depNameTemplate": "defenseunicorns/zarf",
```
Comment (zachariahmiller) [inline renovate.json:48]: just a thought on this. Do we add the settings.json in this repo for the schema validation and then add the match sting we use on that for uds-core for zarf?

I realize we sort of end up "dependent" on zarf version still in that case, but its just for that one scenario. Alternatively, uds could include the zarf schema in its repo and mitigate the need.

**Review by zachariahmiller — COMMENTED** [review]
See comment. Depending on if/how we address that otherwise lgtm

File: renovate.json (hunk 1 above)
Comment (mjnagel) [reply renovate.json:48]: 🤔 currently we don't have the schema so not an issue in uds common tasks but its a good callout. Probably need to pull the zarf schema from uds-cli to handle if we want to drop dependency completely.

Maybe we could also add a task for `uds zarf prepare lint`, which should validate against the correct schema (noting that it doesn't actually handle the IDE feedback issue).

Comment (mjnagel) [reply renovate.json:48]: Going to resolve this thread, think it makes sense to follow up once the above uds-cli issue is resolved.

**Review by MxNxPx — APPROVED** [review]

**Pull request merged**